CHANGELOG.md

```diff
@@ -14,9 +14,48 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
-- Use right interpolation method in WinCLIP resize (<https://github.com/openvinotoolkit/anomalib/pull/1889>)
+### New Contributors
+
+**Full Changelog**:
```

Comment:
Not sure if we need these under `unreleased`. Previously we only had `added`, `changed`, `deprecated` and `fixed`.